noaa/challenges/default.json

```diff
@@ -0,0 +1,82 @@
+    {
+      "name": "append-no-conflicts",
+      "description": "Indexes the whole document corpus using Elasticsearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Rally will only start the benchmark if the cluster turns green and we want to ensure that we don't use the query cache. Document ids are unique so all index operations are append only. After that a couple of queries are run.",
+      "default": true,
+      "index-settings": {
+        "index.number_of_shards": 1,
+        "index.number_of_replicas": 0,
+        "index.queries.cache.enabled": false
+      },
+      "schedule": [
+        {
+          "operation": "index",
+          "#COMMENT": "This is an incredibly short warmup time period but it is necessary to get also measurement samples. As this benchmark is rather about search than indexing this is ok.",
+          "warmup-time-period": 10,
```

Comment:
Is this short warmup time period warranted here? I think this is only necessary for percolator (where indexing throughput is not interesting anyway). Ideally we'd have at least 240 seconds here.